_data/collection-index.yml

```
@@ -568,3 +568,8 @@
   contact: https://github.com/schlich/cst-devcontainer-feature/issues
   repository: https://github.com/schlich/cst-devcontainer-feature
   ociReference: ghcr.io/schlich/cst-devcontainer-feature
+- name: LINO PIMO SIGO Toolbox
+  maintainer: CGI France
+  contact: https://github.com/CGI-FR/lino-devcontainer/issues
+  repository: https://github.com/CGI-FR/lino-devcontainer/
+  ociReference: ghcr.io/lino-devcontainer/lino
```

```suggestion
  ociReference: ghcr.io/cgi-fr/lino-devcontainer
```
Hi 👋 

The `ociReference` here refers to the collection metadata artifact instead of the individual Features. Hence, the author needs to only add a single entry irrespective of the number of Features that might exist in their repository.

Can you help update this? thanks!

---

Thank you for pointing out this mistake.